tests/everest/test_everserver.py

```diff
@@ -244,42 +192,22 @@ def mocked_server(url, verify, auth, timeout, proxies):
     # The server should fail and store a user-friendly message.
     assert status["status"] == ServerStatus.failed
     assert OPT_FAILURE_REALIZATIONS in status["message"]
-    assert "job1 Failed with: job 1 error 1" in status["message"]
-    assert "job1 Failed with: job 1 error 2" in status["message"]
-    assert "job2 Failed with: job 2 error 1" in status["message"]
 
```

Comment:
Do we not have this information about the failure anymore?

Reply:
We will if it exists in the snapshot, but the main difference here from before is that we dont read the error file from disk, as that relies more on the shared file system. If the information is in the snapshot, we will have it. If a job fails, the monitoring will give out whatever information is in the snapshot, so in this specific instance, the the snapshot property `error` had: `job 2 error 1`, that would be given. It would still be possible to read the error file and get the full error, but the reason for example ert does not do that, and the reason it is not in the snapshot is because it can be quite large. We could potentially `tail` the file, and it would be smaller. But that should be done on the server side then.